### PR TITLE
feat: 404 page

### PR DESCRIPTION
**Pull request by Pratham-Mishra178**

### CODE REVIEW

**Comment by ashuvssut** [comment]
image

File: pages/404/404.tsx [hunk 1]
```diff
@@ -35,6 +33,7 @@ export const ErrorMessage: NextPage = () => {
 						pt={7}
 						pb={4}
 						mx={1}
```
Comment (ashuvssut) [inline pages/404/404.tsx:35]: remove all these system props.

use h1 variant of Typography instead

**Review by ashuvssut — CHANGES_REQUESTED** [review]

**Comment by Pratham-Mishra178** [comment]
> @Pratham-Mishra178 image

i have used system props to give padding betwwen two texts

**Comment by ashuvssut** [comment]
Please always check what changes I make to your PR.

As a maintianer I can commit changes to your forked remote repo. you should always pull my changes from your forked remote repo to check what changes I have made locally on your machine.

For, now I have removed stylings from your page. there was a lot of bad styles that made your page unresponsive.
mainly, you used media-queries. do not use media-queries. only use wherever its an absolute necessity.

I hope you have checked my changes that I made to your Previously merged PR

**Comment by Pratham-Mishra178** [comment]
> Please always check what changes I make to your PR.
> 
> As a maintianer I can commit changes to your forked remote repo. you should always pull my changes from your forked remote repo to check what changes I have made locally on your machine.
> 
> For, now I have removed stylings from your page. there was a lot of bad styles that made your page unresponsive. mainly, you used media-queries. do not use media-queries. only use wherever its an absolute necessity.
> 
> I hope you have checked my changes that I made to your Previously merged PRs. Learn from those changes. else ask me why I made those changes

yes I have understand this . Thank You!

**Pull request merged**